.github/workflows/ci.yml

```diff
@@ -3,7 +3,16 @@ on: [pull_request, pull_request_target, push]
 
 jobs:
   lint-ui:
-    if: github.event_name == 'push'
+    if: >-
+      (
+        github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name == github.repository
+      ) || (
+        github.event_name == 'pull_request_target' &&
+        github.event.pull_request.head.repo.full_name != github.repository
+      ) || (
+        github.event_name == 'push'
+      )
```

Comment:
@ariefrahmansyah I added these conditions back because I think we should be running these jobs on the upstream repository as well (to have it shown in the PR view). It is difficult to track if we only run this on the forked repository (won't be able to notice if these jobs failed).

Reply:
Noted. LGTM